src/js/components/help/help.js

```diff
@@ -63,24 +46,15 @@ const eachRecursive = (obj, path, level, accu, isHosted, spacing) =>
         path: this_path,
         hosted: value.hosted,
         style: { paddingLeft: `calc(${level} * ${spacing})` },
-        exact: true
+        exact: true,
+        url: value.url ? value.url : ''
```

Comment:
you could add a  `secondaryAction: value.url ? <LinkIcon /> : null,` here as well, then the extra icon at the end would be handled by the `LeftNav`/`ListItem` 👌  (this is what is used [here in the settings](https://github.com/mendersoftware/gui/blob/master/src/js/components/settings/settings.js#L94))